sqlglot/dialects/tsql.py

```diff
@@ -158,6 +158,7 @@ class Tokenizer(Tokenizer):
             "SQL_VARIANT": TokenType.VARIANT,
             "NVARCHAR(MAX)": TokenType.TEXT,
             "VARCHAR(MAX)": TokenType.TEXT,
+            "APPLY": TokenType.APPLY,
```

Comment:
really sorry about the swirl, but i think we can add this to the generic tokenizer but we neeed to in the parser add 

APPLY to ID_VAR_TOKENS and perhaps TABLE_ALIAS_TOKENS like NATURAL

Reply:
No problem, moved it back to the generic tokenizer.